proxy/http/remap/PluginFactory.cc

```diff
@@ -0,0 +1,268 @@
+/** @file
+
+  A brief file description
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+/**
+ * @file PluginFactory.cc
```

Comment:
Didn't you do this in line 3? Also, there's no point in putting the file name in a doxygen comment, doxygen knows what the file name is.

Reply:
Sure, will merge them (see my previous comment). 

Also will remove the name from `@file` although I do not mind specifying the `@file` name explicitly. The reason is that [the Doxygen doc about the structural commands](http://www.doxygen.nl/manual/docblocks.html#structuralcommands) feels somewhat unclear, also many of the Doxygen examples contain the `@file` name explicitly (i.e. see `structcmd.h` example right after the "Attention" note).